### PR TITLE
NCSDK-3416: fix gs assistant installation information

### PR DESCRIPTION
**Pull request by greg-fer**

### CODE REVIEW

File: doc/nrf/getting_started.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Getting started
 ###############
 
-To quickly get started with the |NCS|, use the `Getting Started Assistant <Getting Started Information_>`_ to set up your development environment.
+To quickly get started with the |NCS|, check the :ref:`gs_assistant` section to set up your development environment.
```
Comment (ru-fu) [inline doc/nrf/getting_started.rst]:
```suggestion
To quickly get started with the |NCS|, use the :ref:`Getting Started Assistant <gs_assistant>` to set up your development environment.
```

File: doc/nrf/gs_assistant.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
+.. _gs_assistant:
+
+.. |GSA| replace:: Getting Started Assistant
```
Comment (ru-fu) [inline doc/nrf/gs_assistant.rst]: This should go into the shortcuts.txt file.

File: doc/nrf/gs_assistant.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+.. gs_assistant_intro
+
+Installing the |NCS| with |GSA|
```
Comment (ru-fu) [inline doc/nrf/gs_assistant.rst]:
```suggestion
Installing the |NCS| with the |GSA|
```

File: doc/nrf/gs_assistant.rst [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+###############################
+
+The |GSA| application provides you with step-by-step instructions for installing the toolchain for |NCS|.
+
+The application allows you to check which of the required tools are already installed on your computer and verify the installation.
+It also guides you through installing the parts of the toolchain that you are missing.
+In addition, it will help you clone the |NCS| repositories and set up your first project in |SES|.
+
+.. gs_assistant_installing-gsa
```
Comment (ru-fu) [inline doc/nrf/gs_assistant.rst]: All of these need a : at the end and a _ at the beginning (or they will just be comments, but not IDs that can be linked to).

File: doc/nrf/gs_assistant.rst [hunk 5]
```diff
@@ -0,0 +1,85 @@
+.. _gs_assistant:
+
+.. |GSA| replace:: Getting Started Assistant
+
+.. gs_assistant_intro
```
Comment (ru-fu) [inline doc/nrf/gs_assistant.rst]: Not needed (since you have _gs_assistant at the top of the file).

File: doc/nrf/gs_assistant.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+.. gs_assistant_installing-ncs
+
+Installing the |NCS|
+********************
+
+Once you installed |GSA|, open it in nRF Connect for Desktop.
+The |GSA| app will open with the Overview section displayed in the main window.
+
+.. figure:: images/gs-assistant_overview.PNG
```
Comment (ru-fu) [inline doc/nrf/gs_assistant.rst]: I don't think this screenshot is needed (people will see it when they open the app).

File: doc/nrf/gs_assistant.rst [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+Next to each step you can use one or both of the following buttons:
+
+- :guilabel:`Mark done` -- marks a step as completed.
+- :guilabel:`Verify` -- checks if the installation is correct.
+
+The verification process is reflected in the vertical progress bar animation.
+Once the verification is successful, the vertical progress bar changes color to green and the Log section is updated.
+At the same time, the :guilabel:`Mark done` button changes to :guilabel:`Mark not done`.
+
+.. figure:: images/gs-assistant_UI.gif
```
Comment (ru-fu) [inline doc/nrf/gs_assistant.rst]: nice :)

File: doc/nrf/gs_assistant.rst [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+   The nRF Connect for Desktop main window
+
+The app is installed on your machine, and the :guilabel:`Install` button changes to :guilabel:`Open`.
+
+.. _gs_assistant_installing-ncs:
+
+Installing the |NCS|
+********************
+
+Once you installed |GSA|, open it in nRF Connect for Desktop.
```
Comment (b-gent) [inline doc/nrf/gs_assistant.rst]:
```suggestion
Once you have installed |GSA|, open it in nRF Connect for Desktop.
```

File: doc/nrf/gs_assistant.rst [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+The |GSA| app will open with the Overview section displayed in the main window.
```
Comment (b-gent) [inline doc/nrf/gs_assistant.rst]:
```suggestion
The |GSA| app opens with the Overview section displayed in the main window.
```

File: doc/nrf/gs_assistant.rst [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+   :alt: The Variables section of the |GSA| app main window, cropped
+
+   The Variables section
+
+.. _gs_assistant_installing-ncs_verify:
+
+Verification options
+====================
+
+Next to each step you can use one or both of the following buttons:
```
Comment (b-gent) [inline doc/nrf/gs_assistant.rst]:
```suggestion
Next to each step, you can use one or both of the following buttons:
```

**Comment by b-gent** [comment]
tiny comments from me, otherwise lgtm

**Pull request merged**